docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md

```diff
@@ -0,0 +1,30 @@
+# Run jobs in case of insufficient capacity for region without Run shift policy flag
+
+Test verifies that run isn't restarted in another regions in case of insufficient capacity if initial region doesn't have Run shift policy flag.
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **CLOUD REGIONS** tab
+4. Select `region1` on the left panel
+5. Save the ***Run shift policy*** checkbox state, then untick it if needed
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the Tools page | |
+| 2 | Select test tool | |
+| 3 | At the tool page, hover over the **Run v** button | |
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Select `p2.8xlarge` value for **Node type** | |
```

Comment:
I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

Reply:
Done